spec/README.md

```diff
@@ -0,0 +1,90 @@
+Test Levels
+===========================
+
+This folder contains tests at the unit, acceptance, and integration level for the "puppetlabs-iis" module. The unit 
+tests are kept in the "spec/unit" level folder of the repository. Acceptance in the "spce/acceptance" and Integration at
+the "spec/integration" level.
+
+## Acceptance Folder
+
+At Puppet Labs we define an "acceptance" test as:
```

Comment:
Arguing by commercial authority is also the weakest possible argument we can make here. Focus on the technical aspects: When are those tests used? How are they used? What are they useful for? What are their limitations?